helm/charts/determined/values.yaml

```diff
@@ -41,3 +41,29 @@ checkpointStorage:
   # accessKey: <access_key>
   # secretKey: <secret_key>
   # endpointUrl: <endpoint_url>
+
+# This is the number of GPUs there are per machine. Determined uses this information when
+# scheduling multi-GPU tasks. Multi-gpu (distributed training) tasks will be scheduled as
+# slotsPerTask / slotsPerNode separate pods (tasks sizes that are not divisible by slotsPerNode
+# are never scheduled.
+slotsPerNode:
```

Comment:
question: what happens if the cluster has different instance types? Is there another way we are making sure each node has the same number of GPUs?